sympy/liealgebras/type_a.py

```diff
@@ -133,17 +133,7 @@ def cartan_matrix(self):
         [ 0,  0, -1,  2]])
 
         """
-
-        n = self.n
-        m = 2 * eye(n)
-        i = 1
-        while i < n-1:
-            m[i, i+1] = -1
-            m[i, i-1] = -1
-            i += 1
-        m[0,1] = -1
-        m[n-1, n-2] = -1
-        return m
+        return super().cartan_matrix()
```

Comment:
Is this `TypeA.cartan_matrix` method necessary? Would `Standard_Cartan.cartan_matrix` suffice?

Reply:
The original authors had implemented by hand all the cartan matrices for each of the algebras `TypeA` - `TypeG`. With each method, they included some good documentation for each `cartan_matrix` method. I wanted to preserve their documentation on each but move the actual implementation to  `StandardCartan`.

Reply:
The documentation for these could be moved to the class docstring or the rst docs. We probably shouldn't implement methods just for the sake of the docstring.